package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "react-native-zoom-us",
-  "version": "4.0.4",
+  "version": "5.0.0",
```

Comment:
do not update version in pr. I run `npm update major` on master and it does some post-update work